activesupport/lib/active_support/core_ext/pathname/blank.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "pathname"
+
+class Pathname
+  # An Pathname is blank if it's empty:
+  #
+  #   Pathname.new("").blank?      # => true
+  #   Pathname.new(" ").blank?     # => false
+  #   Pathname.new("test).blank?  # => false
```

Comment:
missing an `"`? (should be `Pathname.new("test").blank?`)

Reply:
Thanks, fixed in https://github.com/rails/rails/commit/c9c1768a91c2b6e6ca68268ed87b3a7327ab81dd